org.jacktrip.JackTrip.yml

```diff
@@ -0,0 +1,50 @@
+app-id: org.jacktrip.JackTrip
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: jacktrip
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  # - --socket=wayland
+  # Needs network access
+  - --share=network
+  # Pipewire/Jack
+  - --filesystem=xdg-run/pipewire-0
+  # For setting realtime priority for network thread?
+  - --socket=system-bus
+cleanup:
+  - /lib/python3.8
+  - /share/man
+modules:
+  - shared-modules/linux-audio/jack2.json
```

Comment:
you need to had the git submodule for this to work.

Reply:
I was already wondering why this doesn't happen on my machine. One git reset too many.